api-explorer/README.md

```diff
@@ -0,0 +1,10 @@
+# Hosted API Explorer
+
+This directory contains [swagger-ui](https://github.com/swagger-api/swagger-ui) assets that serve as a hosted API Explorer for LoopBack 4 applications. The UI will be available at https://loopback.io/api-explorer?url=<swagger-spec-url>.
```

Comment:
I think you should use different characters than `<` and `>` here, because angle brackets are interpreted as HTML markdown, effectively hiding `<swagger-spec-url>` part from the URL.  See [the rendered README](https://github.com/strongloop/loopback.io/blob/3c7c493e56e354dae2862d2e3f1301d85a5989f8/api-explorer/README.md).